src/App.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
+import Tabs from './components/Tabs';
 
-import './App.scss';
+import './scss/App.scss';
```

Comment:
```suggestion
import './styles/App.scss';
```
в таком случае, но вообще файл со стилями к компоненту принято хранить рядом с ним